docs/adr/0013-gestion-erreurs-api-ihm.md

```diff
@@ -0,0 +1,213 @@
+             event.preventDefault();
+             try {
+               await this.model.save();
+               this.notifications.success('L’organisation a été créée avec succès.');
+               this.transitionToRoute('authenticated.organizations.get', this.model.id);
+             } catch (error) {
+               this.notifications.error('Une erreur est survenue.');
+             }
+           }
+   
+- Afficher un message générique à l'utilisateur final pour tous les cas d'erreur peut le perturber et le diriger sur une mauvaise cause de l'erreur. 
+ 
+ Exemple: Le message 'identifiant incorrect' est affiché lors d'une indisponibilité temporaire de l'application a déjà poussé l'utilisateur à réinitialiser son mot de passe.
+ 
+- Se baser sur le code HTTP, pour différencier les erreurs.
+ 
+       _manageErrorsApi(firstError) {
+            const statusCode = _.get(firstError, 'status');
+            this.errorMessage = this._showErrorMessages(statusCode);
+          }
+        
+          _showErrorMessages(statusCode) {
+            const httpStatusCodeMessages = {
+              '400': API_ERROR_MESSAGES.BAD_REQUEST.MESSAGE,
+              '401': API_ERROR_MESSAGES.LOGIN_UNAUTHORIZED.MESSAGE,
+              '500': API_ERROR_MESSAGES.INTERNAL_SERVER_ERROR.MESSAGE,
+              '502': API_ERROR_MESSAGES.INTERNAL_SERVER_ERROR.MESSAGE,
+              '504': API_ERROR_MESSAGES.GATEWAY_TIMEOUT.MESSAGE,
+              'default': API_ERROR_MESSAGES.INTERNAL_SERVER_ERROR.MESSAGE,
+            };
+            return this.intl.t(httpStatusCodeMessages[statusCode] || httpStatusCodeMessages['default']);
+          }
+  
+**Point 5:** Dans le cas de la PR https://github.com/1024pix/pix/pull/1727, nous souhaitons afficher des messages appropriés selon le scénario X. Ce qui nous amène à avoir plusieurs messages d'erreurs pour le même status HTTP.
+Détail: Le message à afficher est calculé côté IHM à cause de l'internationalisation. Pour ce faire, l'IHM récupère des informations non présentes côté client via l'erreur renvoyée par l'API. 
+   
+## Décision
+
+**Point 1:**
+- Utiliser la structure suivante partout dans le code: 
+
+            - status: the HTTP status code applicable to this problem, expressed as a string value.
+            - code: an application-specific error code, expressed as a string value.
+            - title: a short, human-readable summary of the problem that SHOULD NOT change from occurrence to occurrence of the problem, except for purposes of localization.
+
+            Exemple pour les `BadRequest` :
+            const jsonApiError = new JSONAPIError({
+              status: errorHttpStatusCode.toString(),
+              title: 'Bad request',
+              detail: err.details[0].message,
+            });
+
+**Point 2:**
+
+- Renvoyer la violation JOI dans l'erreur:
+Exemple: `detail: err.details[0].message` au lieu de `detail: 'The server could not understand the request due to invalid syntax.'`
```

Comment:
Tu peux rajouter un lien vers une implémentation "correcte", ça donne du contexte.
Et je pense qu'il n'est pas nécessaire de mentionner l'ancienne solution 
`au lieu de detail: 'The server could not understand the request due to invalid syntax.`